spanner/api/Spanner.Samples/AddAndDropDatabaseRole.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_add_and_drop_database_role]
+
+using Google.Cloud.Spanner.Data;
+using System.Threading.Tasks;
+
+public class AddAndDropDatabaseRoleSample
+{
+    public async Task AddDatabaseRoleAsync(string projectId, string instanceId, string databaseId, string databaseRole)

```

Comment:
I have seen that the region tag contains both add and drop. However, we have two samples in one file. (style guide says one sample per file) so may be worth discussing with the spanner team to update the region tag to adhere to the sample style guide.

Reply:
Its already discussed with the spanner team. This sample is part of some documentation and add and drop roles comes under the single heading that's why they require it to be single sample.